relayer/.env.example

```diff
@@ -1,4 +1,13 @@
 KEY_PAIR=[4, 124, 192, 239, 178, 129, 148,  57,  40, 119,  77, 194, 134,  21,  20, 122, 220, 120, 155,   8,  57, 232, 42,  53,  51,  62,  55,   1, 204, 203, 196, 236, 204, 78,  17,   6,  95, 154, 116,  29,  29,  93,  86, 181, 36, 122,  27,  66, 241, 193, 198, 240, 239, 184, 137, 168, 109,  45, 184, 210, 190,   8, 136,  44]
 RELAYER_RECIPIENT=[ 79, 149, 114, 205, 153,  39, 179,  25,  52, 107, 164, 75, 124, 102,  73,  91, 112,  61, 152,  70, 250, 208, 134,  24, 159, 176,  52, 177, 183,  99, 151, 146, 140, 232,  23, 147,  47,  68, 206,  56, 135, 249,  23,  34, 252, 245, 228,  51, 240,  38, 154, 200, 137,   2, 166, 173,  64, 131, 141,  14, 150, 134,  80,  13 ]
-LOOK_UP_TABLE=""
-TEST_ENVIROMENT=true
\ No newline at end of file
+LOOK_UP_TABLE="4Dcx88YhY6YD4ojbGtK1e1x344WPhurHM6GxX1JCDkAU"
+TEST_ENVIRONMENT=true
+ENVIRONMENT=LOCAL
+NETWORK=LOCAL
+RPC_URL="http://localhost:8899"
+# DATABASE_URL=${mainnet-testing-v1.DATABASE_URL}
```

Comment:
Why is it commented?

Reply:
I suppose this is the url we will use with a separate digital ocean database instance which is not tested yet.
Cc @SwenSchaeferjohann

Reply:
yup

Reply:
yes

Reply:
I will add more detailed inline comments for this file in a separate PR.